### PR TITLE
Update omniscien.md

### PR DESCRIPTION
**Pull request by dionwiggins**
Added current languages and updated self service information which was not correct.

# Description

Fixes # _[Add the issue number]_

## Type of PR

- Creates the article _[article name]_
- Edits _[article name]_
- Other _[please describe]_

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).

### CODE REVIEW

**Comment by cefoo** [comment]
Hi @dionwiggins!
Thanks for your PR!

**Comment by bittlingmayer** [comment]
By the way, note that your website lists the wrong language code for Czech.

https://omniscien.com/machine-translation/supported-languages/

> Czech (CZ / cs-CZ)

Should be `cs`.

File: apis/omniscien.md [hunk 1]
```diff
@@ -137,7 +169,7 @@ supported_languages:
   base_code: ja
   name: Japanese
   variant_name: null
-- slug: khmer
+- slug: georgian
```
Comment (bittlingmayer) [inline apis/omniscien.md:172]: Seems like a typo.

File: apis/omniscien.md [hunk 2]
```diff
@@ -149,6 +181,18 @@ supported_languages:
   base_code: ko
   name: Korean
   variant_name: null
+- slug: kurdish
+  code: ku
+  normalized_code: kk
```
Comment (bittlingmayer) [inline apis/omniscien.md:186]: Seems like a mistake.  kk is Kazakh.

Comment (dionwiggins) [reply apis/omniscien.md:186]: Corrected. Thank you.

File: apis/omniscien.md (hunk 1 above)
Comment (dionwiggins) [reply apis/omniscien.md:172]: Corrected. Thank you.

**Comment by dionwiggins** [comment]
> By the way, note that your website lists the wrong language code for Czech.
> 
> https://omniscien.com/machine-translation/supported-languages/
> 
> > Czech (CZ / cs-CZ)
> 
> Should be `cs`.

Corrected. Thanks for the spot.

**Pull request merged**